realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -975,6 +1022,37 @@ void BaseRealSenseNode::publishFrame(
     }
 }
 
+
+void BaseRealSenseNode::publishRGBD(cv::Mat& rgb_cv_matrix, cv::Mat& depth_cv_matrix, const rclcpp::Time& t)
+{
+    if (_rgbd_publisher && 0 != _rgbd_publisher->get_subscription_count())
+    {
+        ROS_DEBUG_STREAM("Publishing RGBD message");
+        unsigned int rgb_width = rgb_cv_matrix.size().width;
+        unsigned int rgb_height = rgb_cv_matrix.size().height;
+        unsigned int rgb_bpp = rgb_cv_matrix.elemSize() ? rgb_cv_matrix.elemSize() : 3;
```

Comment:
`rgb_cv_matrix.elemSize()` can be 0?

Reply:
it was only for debugging, removing this